front/public/locales/en/operationalStudies/scenario.json

```diff
@@ -82,8 +82,11 @@
     "validityFilter": "Trains validity"
   },
   "toggleTimetable": "Toggle the timetable",
-  "trainCount_other": "{{ count }} out of {{ totalCount }} trains selected",
-  "trainCount_zero": "{{ totalCount }} trains",
+  "trainCount_one": "1 out of $t(train, { 'count': {{totalCount}} }) selected",
```

Comment:
Nit: do we really need the `trainCount_one` special case? Wouldn't this be taken care of by `trainCount_other`?

Reply:
This is what I tried first but then something... peculiar happened
[Screencast_20241025_161918.webm](https://github.com/user-attachments/assets/c41dbb06-e29c-4508-b034-1a606ba38379)

Right then and there I decided I would save myself a headache by just being a bit more exhaustive

Reply:
Oh my.

One less headache sounds good to me!